_posts/2021-11-08-hhvm-4.136.markdown

```diff
@@ -0,0 +1,23 @@
+---
+title: "HHVM 4.136"
+layout: post
+author: atry
+category: blog
+---
+
+HHVM 4.136 is released! HHVM 4.131&ndash;4.135 remain supported, as do the 4.102 and 4.128 LTS releases.
+
+# Highlights
+
+- Add quickfix for missing interface methods.
+- [The `readonly` feature](https://docs.hhvm.com/hack/experimental-features/read-only) is no longer considered unstable.
+
+# Breaking Changes
+
+- Methods with a `private final` modifier are not supported any more. Use `private` instead.
+- The `partial` typing mode is removed. This includes both support for `<?hh // partial` in .php files, and for .hackpartial files. The `--disable-modes` flag, which was used to treat partial files as strict ones, is no longer available because it is now the default behavior.
+- Raise an error when a trait is `<<__ConsistentConstruct>>` but does not declare an explicit constructor.
+
+# Future Changes
+
+- The `readonly` modifier is now available in the reflection API via the `isReadonly()` or `returnsReadonly()` functions.
```

Comment:
Is now or will be? If it is, this shouldn't be under 'future changes' and we can skip this section if nothing else goes in it.